run.go

```diff
@@ -117,7 +117,7 @@ func updateConfigWithFlags(rootCmd *cobra.Command, cnfg *config.Config, programA
 	subCmd, flags, _ := rootCmd.Find(programArgs)
 	// fall back to cmd.Help, even if there is error
 	_ = subCmd.ParseFlags(flags)
-	// initialize config from file
+	//initialize config from file
```

Comment:
This change is not correct, there should be a space after `//`.

Reply:
That was a typo, fixed. [9ae1d29](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/9ae1d29bf597e5540d6efe1c467ed614740bdb24)